packages/clerk-js/src/ui/common/constants.ts

```diff
@@ -96,6 +96,10 @@ export const WEB3_PROVIDERS: Web3Providers = Object.freeze({
     id: 'coinbase',
     name: 'Coinbase Wallet',
   },
+  coinbase_wallet: {
```

Comment:
🙃 SW = smart wallet
```suggestion
  coinbase_sw: {
```

Reply:
Coinbase prefer to use the term Coinbase Wallet rather than Coinbase Smart Wallet